RELEASING.md

```diff
@@ -4,11 +4,13 @@ Outline for releasing this package to Hackage and Stackage
 
 ## Steps
 
-* Bump version in `package.yaml` in this repository. Create a PR with this change.
+* Bump version in `package.yaml` in this repository. Create and merge a PR with this change.
 * `git tag <version> -m "Release version <version>"`
 * `git push origin --tags`
-* `stack sdist`
-* Copy to a folder that can be navigated to for upload
-* Upload here: http://hackage.haskell.org/packages/upload
-* `scripts/hackage-docs.sh <user>`
+* Build and upload to Hackage:
+   * Make sure your workspace is clean
+   * `stack sdist`
+   * Copy to a folder that can be navigated to for upload
+   * Upload here: http://hackage.haskell.org/packages/upload
```

Comment:
@dfithian Are these steps preferred to `stack upload` for some reason? The result seems to be the same (but I haven't tried to do `stack upload` and then also upload docs.)

Reply:
Not at all, those are just the instructions I originally wrote down so are the ones I follow now.